packages/waku/src/router/client.ts

```diff
@@ -321,3 +322,34 @@ export function Router() {
     createElement(InnerRouter),
   );
 }
+
+function notAvailableInServer(name: string) {
+  return () => {
+    throw new Error(`${name} is not in the server`);
+  };
+}
+
+/**
+ * ServerRouter for SSR
+ * This is not a public API.
+ */
+export function ServerRouter({
+  children,
+  loc,
+}: PropsWithChildren<{ loc: ReturnType<typeof parseLocation> }>) {
```

Comment:
Just my preference:
```suggestion
}: { children: ReactNode; loc: ReturnType<typeof parseLocation> }) {
```

Reply:
I'll revert this since it would force us to pass children and children would force us to have keys if we pass array, unless we pass a fragment, which does not look good too.

Reply:
It's just a TS issue, right?
I can take care of it later.

Reply:
hmm, ts issue, key issue + code formatting issue